eng/Packages.Data.props

```diff
@@ -98,6 +98,7 @@
     <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <PackageReference Update="System.Threading.Channels" Version="4.7.1" />
     <PackageReference Update="System.Security.Cryptography.ProtectedData" Version="4.7.0" />
+    <PackageReference Update="System.Security.Cryptography.Cose" Version="7.0.0" />
```

Comment:
Please bump this up a line to preserve alphabetical order.   If you don't mind moving both threading packages to after the System.Text packages, that would be much appreciated.

Reply:
@KrzysztofCwalina, @tg-msft:  Can you confirm that you've approved this dependency?

Reply:
Yes - see the "Re: .NET SDK - addition of System.Security.Cryptography.Cose to Packages.Data.props" thread.